server/fleet/service.go

```diff
@@ -539,8 +539,15 @@ type Service interface {
 	ListMDMAppleEnrollmentProfiles(ctx context.Context) ([]*MDMAppleEnrollmentProfile, error)
 
 	// GetMDMAppleEnrollmentProfileByToken returns the Apple enrollment from its secret token.
+	// TODO(mna): this may have to be removed if we don't end up supporting
+	// manual enrollment via a token (currently we only support it via Fleet
+	// Desktop, in the My Device page). See #8701.
```

Comment:
Note that I haven't updated #8701 to mention this yet, as maybe we already know that we won't be needing this and if so I'll delete the code immediately as part of this PR. Otherwise I'll make a note in the cleanup ticket.

Reply:
I updated the ticket to mention removing this if unneeded.